src/Components/Counter.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import Button from './Button';
+
+const Counter = ({ count, onClick }) => (
+  <Button onClick={() => onClick(1)}>
```

Comment:
Counter를 사용할 떄 이미 `<Counter onClick={() => onClick(1)} />`으로 하고 있으니 여기에서는 다음과 같이 하는게 좋을 것 같아요
```suggestion
  <Button onClick={onClick}>
```